### DISCUSSION TITLE
Plugin: Session recording á la asciinema

### DISCUSSION THREAD

**Discussion by moritzdietz** [zellij-org/zellij#537] [Ideas]
As many others I appreciate the ability to record my terminal for whatever reason - may it be to revisit an installation of software I did a year ago and go see what I've done, or for tutorial reasons, etc

[Asciinema](https://github.com/asciinema/asciinema) has served well int the past years in combination with tmux. Start an asciinema recording and then start tmux. Now all of the panes and windows of tmux are in the recording.
Asciinema hasn't seen much love over the last years. There was an attempt to port it to Golang by the maintainer which seems to be abandoned by now.
I know that the folks at [Teleport](https://github.com/gravitational/teleport) have a cabability in their product to record sessions for auditing purposes. It strikes me as odd that they haven't spun out the recording stuff into its own product.

Anyhow: I think with the feature plan of zellij (web support, session sharing) it doesn't seem to be far off recording those session.  What do you all think?

**Comment by teamgroove** [comment]
Quite some time passed along since, but: 

<3 It's already possible with tmux involved: https://gist.github.com/worldofprasanna/1861b182103cef452ec58471679a7e5b
